drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -504,6 +504,24 @@ def self.available_units_for_enrollment(project, household_id: nil)
         end
     end
 
+    def self.available_units_for_enrollment(project, household_id: nil)
+      return [] unless project
+
+      # use picklist that includes all available units including units of other types
+      picklist = admin_available_units_for_enrollment(project, household_id: household_id)
+      return picklist unless household_id # no household, so no need to filter unit types
+
+      # drop units that have different types
+      hh_unit_type_ids = project.enrollments.where(household_id: household_id).map(&:current_unit_type).compact.map(&:id).uniq
+      return picklist unless hh_unit_type_ids.any? # household doesn't have a unit type, so no need for further filtering
```

Comment:
perhaps more readable?
```suggestion
      return picklist if hh_unit_type_ids.empty? # household doesn't have a unit type, so no need for further filtering
```